README.md

```diff
@@ -1,129 +1,108 @@
+## Development Instructions
+This section discusses the normal development process.  It assumes that you are
+a developer new to the application and steps through the build dependencies as
+well as the actual build process.
 
-	To see if the Cordova/Phonegap plugins are being called by the app, open `index.html` in the devcycle-mobile-app root after running `sencha app build native`. Upon hitting the Pause button, a popup dialog will show the Cordova/Phonegap call and finally the Pause button would go to a Resume button state. If it doesn't work, it is likely that Cordova is not working properly with the project.
+### Dependencies
+The following lists of dependencies *must* be installed prior to building the
+application.
 
-###Dependencies Required for Building Project
-* Git 
+* Git
 * NodeJS (Current Version)
 	* [http://nodejs.org/](http://nodejs.org/ "NodeJS Site")
 * Ruby 1.9.3
-	* (Windows) During installation, Ensure to select "Add Ruby Executables to your PATH"
+  * (Windows) During installation, Ensure to select "Add Ruby Executables to
+    your PATH"
 * Java Runtime Environment > 1.7
 * Java Development Kit 7
 * Python (Latest version)
 * Ant
 	* (Windows) [https://code.google.com/p/winant/](https://code.google.com/p/winant/)
 	* (Mac/Linux/Unix) See [http://ant.apache.org/manual/install.html](http://ant.apache.org/manual/install.html)
-* Cordova 3.6.3-0.2.13 (This version addressed security issues that Google flagged and resulted in removal from the Google Play Store)
-	* Install this version by typing in the terminal 
-		* (Windows) `npm install -g cordova@3.6.3-0.2.13`
-		* (Mac/Linux/Unix) `sudo npm install -g 3.6.3-0.2.13`
+* Cordova 5.4+
+	* Install this version by typing in the terminal
+		* `[sudo] npm install -g cordova@5.4`
 * Sencha Command Line Tools 6.0.2.14 available [here](https://www.sencha.com/products/extjs/cmd-download/)
-* SASS (Ruby Gem)
-	* Type `gem install sass` after ruby has been installed
-* Compass (Ruby Gem)
-	* Type `gem install compass` after ruby has been installed
-	*
 * Android SDK (if building native Android)
-	* Ensure that you add the Android SDK to the PATH with access to the following directories:
+  * Ensure that you add the Android SDK to the PATH with access to the following
+    directories:
 		* platform-tools
 		* tools
 * iOS SDK (if building native iOS)
 
+#### Required Cordova Plugins
+Several cordova plugins are required to build the app.  Running `python
+fetchPlugins.py` from the root directory will install all dependencies.  The
+full list of requirements can be found by opening that script.
+
+### Building the Project
+1. Clone this repository and `cd` into the directory `git clone
```

Comment:
This reads a little weird. If you add a colon before the command or say "using the following command" it might be a little clearer.